CustomCollections/CustomCollections/CustomHashTable.cs

```diff
@@ -0,0 +1,220 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace CustomCollections
+{
+    class CustomHashTable<TKey, TValue> : IDictionary<TKey, TValue>
+    {
+        private static KeyValuePair<TKey, TValue>[] _hashTable;
```

Comment:
Why is this static?

Reply:
I forget about this. This must be not static. I fix this.

Reply:
this